src/Test/winswTests/ServiceDescriptorTests.cs

```diff
@@ -1,445 +1,445 @@
-﻿using System;

```

Comment:
Why did the entire file change?

Reply:
Actually I don't know why its preview in this way???

Reply:
This file doesn't have a BOM header in master. I suggest removing the BOM in this PR to reduce diff. (Don't ask me how to.)

Reply:
Opened #590.